server/src/main/java/com/flipkart/varadhi/VaradhiApplication.java

```diff
@@ -97,6 +98,7 @@ private static Future<Vertx> createClusteredVertx(
         EventBusOptions eventBusOptions = new EventBusOptions()
                 .setHost(memberInfo.hostname())
                 .setPort(port)
+                .setClusterPublicHost(memberInfo.address())
```

Comment:
in eventbus options, what is host used for if cluster public host is already present?

Reply:
host will be used for nodeId (internal to vertx) but for actual comms, address will be used.